.travis.yml

```diff
@@ -1,5 +1,5 @@
-dist: trusty
 sudo: required
+dist: xenial
```

Comment:
Does this needs to be added above sudo: required ?
https://docs.travis-ci.com/user/reference/trusty/

I had a similar problem while bumping scikit-learn to 0.15. That's why added dist: trusty